### DISCUSSION TITLE
Censoring and multiple endpoints

### DISCUSSION THREAD

**Discussion by flkloprogge** [nlmixr2/nlmixr2#122] [Q&A]
Hi nlmixr2 development team,

Can nlmixr2 handle censored data for one endpoint only when two endpoints are analysed simultaneously?

Whilst fitting two endpoints simultaneously, where endpoint A does not have censored data and endpoint B has censored (>ULQ) data, I found RUV is much larger for both endpoints compared to when I run endpoint A and B seperately, and a VPC gives me the impression that ULQ handling is somehow applied to both endpoints.

Best regards,
Frank

**Comment by mattfidler** [comment]
Yes, censoring can be applied in multiple endpoint models.

The censoring is applied per observation, so if you see that maybe check your data?
